src/python/py/models/builder.py

```diff
@@ -249,6 +249,17 @@ def __init__(self, config, io_dtype, onnx_dtype, ep, cache_dir, extra_options):
             "output_0": "",             # Output 0 for MLP layer
         }
 
+        # MoE-specific variables
+        self.moe_attrs = {
+            "top_k": 1,                             # Number of experts to select in MoE layer
```

Comment:
Can we set the default values similar to how the default values are set for attention, rotary embeddings, etc? Something like this:

https://github.com/microsoft/onnxruntime-genai/blob/ae18a2cf276ad7800c7e368d313e667b1865c298/src/python/py/models/builder.py#L159-L166

That way, the config's values will be used by default when available.